qiita_pet/templates/study_description_templates/prep_template_info_tab.html

```diff
@@ -0,0 +1,196 @@
+                <tr>
+                  <td id="display-extra-info-{{pt_id}}" colspan="2"></td>
+                </tr>
+              </table>
+            </div>
+          </div>
+        </div>
+        <div class="modal-footer">
+        </div>
+      </form>
+    </div>
+  </div>
+</div>
+
+<!-- Prep Template tab pane -->
+<div class="tab-pane" id="prep_template_info_{{pt_id}}" style="padding: 10px;">
+  <table border="0">
+    <tr>
+      <td>
+        {% if is_editable %}
+          {% module EditInvestigationType(ena_terms, user_defined_terms, pt_id, investigation_type, 0) %}
+        {% else %}
+          <b>Investigation type:</b> {{investigation_type}}
+        {% end %}
+      </td>
+    </tr>
+    <tr>
+      <td style="padding: 5px;">
+        <a class="btn btn-primary" href="/metadata_summary/?study_id={{study_id}}&prep_template={{pt_id}}">
+          Show prep template summary
+        </a>
+      </td>
+    </tr>
+    <tr><td style="padding: 5px;">{% raw current_template_fp %}</td></tr>
+    <tr><td style="padding: 5px;">{% raw current_qiime_fp %}</td></tr>
+    {% if raw_data %}
+      <!-- The prep template has raw data associated -->
+      <tr>
+        <td style="padding: 5px;">{% module RawDataInfoDiv(raw_data, prep_template, study, files) %}</td>
+      </tr>
+      <tr>
+        <td>
+          {% if not preprocessed_data or preprocessing_status.startswith('failed') %}
+            {% if preprocessing_status == 'not_preprocessed' or preprocessing_status.startswith('failed')%}
+              {% if show_preprocess_btn %}
+                <a class="btn btn-primary glyphicon glyphicon-play" data-toggle="modal" data-target="#preprocess-modal-view-{{pt_id}}" style="word-spacing: -10px;"> Preprocess</a>
+              {% else %}
+                <b>{{ no_preprocess_msg }}</b>
+              {% end %}
+            {% end %}
+            <br>
+            <i>Status:</i> {{preprocessing_status}}
+            <br>
+            {% if 'zero-size array' in preprocessing_status %}
+              <b>Your barcodes do not seem to match your prep template info</b>
+            {% end %}
+          {% else %}
+            <b>Preprocessed data generated: </b>
+            <a href="#" onClick="display_tab('preprocessed_data_tab', {{preprocessed_data[0]}});">{{preprocessed_data[0]}}</a>
+            {% for pdi in preprocessed_data[1:] %}
+              , <a href="#" onClick="display_tab('preprocessed_data_tab', {{pdi}});">{{pdi}}</a>
+            {% end %}
+          {% end %}
+        </td>
+      </tr>
+    {% else %}
+      <!-- There is no raw data associated with the prep template, show options for adding raw data -->
+      <tr>
+        <td style="padding: 5px;">
+          <button class="btn btn-primary glyphicon glyphicon-plus" type="button" data-toggle="collapse" data-target="#add-raw-data-{{pt_id}}" aria-expanded="false" aria-controls="add-raw-data-{{pt_id}}" style="word-spacing: -10px;">Add raw data</button>
```

Comment:
Can there be more than one association of raw data per prep template? I thought we agreed it was a 1 to 1 thing, so why have this button here? It should just show the accordion of add new or add from existing, no need for this extra step.